lms/services/annotation_activity_email.py

```diff
@@ -38,7 +38,6 @@ def send_mention(
         email_vars = {
             "assignment_title": assignment.title,
             "course_title": assignment.course.lms_name,
-            "mentioned_user": mentioned_user.display_name,
```

Comment:
The annotation text already includes the mentioned user, so we don't need to display it on its own.